platform_versioned_docs/version-24.3/getting-started/studios.mdx

```diff
@@ -0,0 +1,452 @@
+---
+title: "Studios for interactive analysis"
+description: "An tutorial for creating interactive analysis Studios for Jupyter, RStudio, VSCode, and more"
+date: "17 Feb 2025"
+tags: [platform, studios, data studios, jupyter, rstudio, xpra, vscode, conda]
+toc_max_heading_level: 3
+---
+
+[Studios](../data_studios/index.mdx) allows users to host a variety of container images directly in Seqera Platform compute environments for analysis using popular environments including Jupyter (Python) and RStudio notebooks (R), Visual Studio Code IDEs, and Xpra remote desktops. Each studio session provides a dedicated interactive environment that encapsulates the live environment.
```

Comment:
Add hyperlinks? I didn't know what Xpra was when I first heard it being talked about.